unison-cli/src/Unison/Codebase/Editor/SlurpComponent.hs

```diff
@@ -1,88 +1,102 @@
-{- ORMOLU_DISABLE -} -- Remove this when the file is ready to be auto-formatted
```

Comment:
Ormolu'd everything, see https://github.com/unisonweb/unison/pull/2821/commits/42e10e7ceb3ef3d646c221364f220466141c3642
for the only behavioural changes.